content/applications/sales/point_of_sale/shop/ship_later.rst

```diff
@@ -0,0 +1,44 @@
+==========
+Ship later
+==========
+
+The **Ship Later** feature allows you to sell products and schedule delivery at a later date. This
+feature comes in handy when a product is out of stock, so voluminous that it requires to be
+shipped, or if, for any reason, the customer needs their order to be shipped later, etc.
+
+Configuration
+=============
+
+:ref:`Go to the POS settings <configuration/settings>`, scroll down to the :guilabel:`Inventory`
+section, and enable :guilabel:`Ship Later`.
+
+.. image:: ship_later/settings.png
+   :scale: 85%
+
+Once activated, you can:
+
+- Choose the location from where the products are shipped by completing the :guilabel:`Warehouse`
+  field.
```

Comment:
```suggestion
- Choose the location from where the products are shipped by selecting a :guilabel:`Warehouse`.
```
since you don't type anything but just select it?